package-manifest.toml

```diff
@@ -620,8 +620,8 @@ output.intermediate_only = true
 service_name = "lldp"
 source.type = "prebuilt"
 source.repo = "lldp"
-source.commit = "30e5d89fae9190c69258ca77d5d5a1acec064742"
-source.sha256 = "f58bfd1b77748544b5b1a99a07e52bab8dc5673b9bd3a745ebbfdd614d492328"
+source.commit = "2a9f92dd55817af792670cd86e6a01676f5c79a0"
```

Comment:
This LLDP update gives us a daemon that can understand the new per-port properties.